app/controllers/account_activations_controller.rb

```diff
@@ -1,5 +1,5 @@
 class AccountActivationsController < ApplicationController
-  before_action :get_user, only: [:edit]
+  before_action :find_user, only: [:edit]
   def edit
     if @user && !@user.activated? && @user.authenticated?(:activation, params[:id])
```

Comment:
Seriously?? I thought by now you would have realized how I dislike this design decision.